test/e2e-tests.sh

```diff
@@ -30,8 +30,9 @@ install_pipeline_crd
 failed=0
 
 # Run the integration tests
-header "Running Go e2e tests"
-go_test_e2e -timeout=20m ./test/... || failed=1
+# TODO HACK HACK HACK HACK
```

Comment:
Is this dead code? If not could the comment do a bit more to describe why these lines have been left in but commented out?

Reply:
WHOOPS

Reply:
This is the bit that runs our integration tests! Unfortunately they all run BEFORE the yaml tests so I had temporarily commented them out so I could check if the tests were succeeding without having to wait for these to complete 🤦‍♀️ 

In the long run hopefully we can run them in parallel! The main complication that stops us from doing that right away is that both tests require a boskos cluster